docs/content/how-to/altering-tables.md

```diff
@@ -409,3 +409,51 @@ ALTER TABLE my_table ALTER COLUMN col_a SET DATA TYPE DOUBLE;
 {{< /tab >}}
 
 {{< /tabs >}}
+
+## Adding watermark
+
+The following SQL adds a watermark with strategy `ts - INTERVAL '1' HOUR` on column `ts` which is marked as event time attribute of table `my_table`.
+
+{{< tabs "add-watermark" >}}
+
+{{< tab "Flink" >}}
+
+```sql
+ALTER TABLE my_table ADD WATERMARK FOR ts AS ts - INTERVAL '1' HOUR
```

Comment:
Can you also add an example: `add computed column` here? Sometimes watermark is closely related to computed column.